packages/helm-charts/blockscout/values.yaml

```diff
@@ -314,3 +320,6 @@ blockscout:
   domainName: celo-testnet.org
   gcp:
     projectId: celo-testnet-production
+
+affinity: {}
```

Comment:
Wrong indentation. Looking at the [template](https://github.com/celo-org/celo-monorepo/pull/10209/files#diff-06961cdf1f8a0aefca30f5075472e255d9a6a52449f7c89148cc4af4e93bb8e4R179) you need it to be under `blockscout`. Let's maybe also put it under some meaningful name like `k8s` or `k8sDefaults` or something, now it's a little hard to understand the scope of these values.

Reply:
Thanks! Indeed I forget to save this file 🙃 .

Regarding Values "refactor", when moving to `charts` repository, another task we do is formatting the values, document and generate a Readme, etc... I think that moment is a good opportunity to current structure and adapt to something that makes more sense (also from a documentation/organization point of view).